x-pack/plugins/fleet/server/services/data_streams.ts

```diff
@@ -8,7 +8,7 @@
 import type { IndicesDataStream, IndicesIndexTemplate } from '@elastic/elasticsearch/lib/api/types';
 import type { ElasticsearchClient } from '@kbn/core/server';
 
-const DATA_STREAM_INDEX_PATTERN = 'logs-*-*,metrics-*-*,traces-*-*,synthetics-*-*,profiling-*-*';
+const DATA_STREAM_INDEX_PATTERN = 'logs-*-*,metrics-*-*,traces-*-*,synthetics-*-*,profiling-*';
```

Comment:
BTW, this pattern is used to query for data streams so that they show in `Fleet > Data streams`

do you know if this is something profiling wants? if profiling data streams/indices should be viewed and managed elsewhere, maybe we need to remove it here

Reply:
So far we considered this OK. But trying to get more opinions from the team...

Reply:
ok, glad that it was considered already. this is an easy change to make down the road